src/lib/bootloader/udev_mapping.rb

```diff
@@ -80,8 +80,9 @@ def udev_to_kernel(dev)
       return dev if Yast::Mode.config
 
       devices = Y2Storage::BlkDevice.all(staging)
+      # removes possible multipath wires from the list since they should not appear as disk devices
```

Comment:
Hmm, this explanation looks wrong for me. This method is only about translating udev device received to kernel name. Only reason I see to filter it out is that this multipath wires have identical udev device as its mutlitpath device and we want to translate that udev device to multipath device and not wire. So if I am true, this should be described here.

Reply:
do you think that the following comment fits better?
"As wires devices have identical udev devices as its multipath device, we remove possible multipath wires from the list since we want to translate udev device to multipath device and not wire"

Reply:
yes, that is perfect one.